### DISCUSSION TITLE
[Q&A] pg_restore

### DISCUSSION THREAD

**Discussion by mattdesmarais** [oguimbal/pg-mem#167] [Q&A]
I'm wondering if there's a way to restore postgres dumps in native format:
➜  database file my.dump 
my.dump: PostgreSQL custom database dump - v1.14-0

I'm currently spinning up a docker container on demand, executing `pg_restore`, querying and getting all the data i need and then tearing down the container. This is inefficient and too many moving parts IMHO. 

`	RestoreDBCmd := types.ExecConfig{
		AttachStdout: true,
		AttachStderr: true,
		Cmd:          []string{"pg_restore", "-U", "postgres", "-v", "-d", s.Token, dbDump},
	}
`

Neat Project BTW.

**Comment by oguimbal** [comment]
Thanks :)

Nope, I think that is a bit too low level for pg-mem :/
And that probably wouldnt be easy to implement
